roles/pulp3/tasks/install.yml

```diff
@@ -66,6 +66,14 @@
         virtualenv: '{{ pulp_install_dir }}'
         virtualenv_command: '{{ pulp_python_interpreter }} -m venv'
 
+    - name: Install the psycopg python package
+      pip:
+        name: psycopg2-binary
+        state: present
+        virtualenv: '{{ pulp_install_dir }}'
+        virtualenv_command: '{{ pulp_python_interpreter }} -m venv'
+      when: pulp_install_postgresql
```

Comment:
This variable is whether or not to install  postgresql, but your use is whether to use postgresql. For instance, on travis, we do not want to install postgres (its already installed), but we do still need to install psycopg. Please add another variable, maybe "pulp_db_type" or something.

Reply:
Is there a way I can limit the options of `pulp_db_type`? Or should I not worry about validating it?